README.md

```diff
@@ -1,10 +1,63 @@
 ## angular-filemanager
 File manager developed with AngularJS and Bootstrap by [Jonas Sciangula Street](https://github.com/joni2back)
 
+### use in your project
+
+#### 1) install and use
+1) bower install --save angular-filemanager
+2) include the dependencies in your project:
+
+    <!-- third party -->
+    <script src="/bower_components/angular-translate/angular-translate.min.js"></script>
+    <script src="/bower_components/angular-cookies/angular-cookies.min.js"></script>
+    
+    <!-- angular-filemanager -->
+    <link rel="stylesheet" href="/bower_components/angular-filemanager/dist/angular-filemanager.css">
+    <script src="/bower_components/angular-filemanager/dist/angular-filemanager.min.js"></script>
+    <script src="/bower_components/angular-filemanager/dist/cached-templates.js"></script>
+3) use the angular directive in your HTML:
+
+    <angular-file-manager></angular-file-manager>
+    
+#### 2) configure
+
+To configure the file manager you can simply add a config to your `config.js`:
+
+    angular.module('yourApp')
```

Comment:
I dont understand this piece of code here
Preserve the default config file (is missing in your pr) (then suggest an easy way to extend it)